server/api.ts

```diff
@@ -776,7 +782,6 @@ router.postAsync("/streamer", ensure.isStreamer, async (req, res) => {
  * Removes the current streamer
  * Params:
  *  - match: the _id of the match
- *  - tourney: identifier for the tournament
```

Comment:
oops

Reply:
oops fixed in bc1e56a1a715dbb05a0259c6d8b85e0afb8d2346